### PR TITLE
Added Coke and Coke Block.

### LINKED ISSUE #30: Coke and Coke Blocks
Since Our Blast Furnace runs on these, we'd better actually have them.

### PR DESCRIPTION
**Pull request by jhrldev**
Coke: 2x Coal burn time

Coke Block: 9x Coke burn time

Closes #30 (I'm not sure if it closes #30 because is missing textures)

### CODE REVIEW

File: src/main/kotlin/com/cout970/magneticraft/block/BlockCoke.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import com.cout970.magneticraft.misc.CreativeTabMg
+import net.minecraft.block.Block
+import net.minecraft.block.material.Material
+
+object BlockCoke : BlockBase(
+        material = Material.ROCK,
+        registryName = "block_coke"), FuelProvider<Block> {
+
+    init {
+        creativeTab = CreativeTabMg
```
Comment (cout970) [inline src/main/kotlin/com/cout970/magneticraft/block/BlockCoke.kt]: The creative tab is specified in the super-class BlockBase, there is no need to duplicate it.

File: src/main/kotlin/com/cout970/magneticraft/item/ItemCoke.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.cout970.magneticraft.item
+
+import com.cout970.magneticraft.config.Config
+import com.cout970.magneticraft.fuel.FuelProvider
+import com.cout970.magneticraft.misc.CreativeTabMg
+import net.minecraft.item.Item
+
+object ItemCoke : ItemBase("coke"), FuelProvider<Item> {
+    init {
+        creativeTab = CreativeTabMg
```
Comment (cout970) [inline src/main/kotlin/com/cout970/magneticraft/item/ItemCoke.kt]: Same issue, duplicated, creative tab is defined in ItemBase

**Review by cout970 — COMMENTED** [review]
Fix the 2 little issues to merge the request.

File: src/main/kotlin/com/cout970/magneticraft/item/ItemCoke.kt (hunk 2 above)
Comment (jhrldev) [reply src/main/kotlin/com/cout970/magneticraft/item/ItemCoke.kt]: Ok will fix that and change the name to Coal Coke (looks better)

**Pull request merged**